apps/web-giddh/src/app/subscription/view-subscription/view-subscription.component.ts

```diff
@@ -168,7 +168,7 @@ export class ViewSubscriptionComponent implements OnInit, OnDestroy {
      * @memberof ViewSubscriptionComponent
      */
     public changeBilling(): void {
-        this.router.navigate(['/pages/subscription/change-billing/' + this.viewSubscriptionData?.billingAccount?.uniqueName]);
+        this.router.navigate(['/pages/subscription/change-billing/' + this.viewSubscriptionData?.billingAccountUniqueName]);
```

Comment:
Suggestion

this.router.navigate([`/pages/subscription/change-billing/${this.viewSubscriptionData?.billingAccountUniqueName}`]);